first_hour_on_vwb/working_with_data.ipynb

```diff
@@ -0,0 +1,577 @@
+    "class ResolveDatasetWidget(object):\n",
+    "    def __init__(self):\n",
+    "        self.table = None\n",
+    "        self.client = bigquery.Client()\n",
+    "        self.label = widgets.Label(\n",
+    "            value=\"Please provide appropriate values in the input boxes.\")\n",
+    "        self.resource_names = self.get_bq_resources()\n",
+    "        self.name_dropdown = wu.DropdownInputWidget(\n",
+    "            self.resource_names, self.resource_names[0], \"Resource Name:\").get()\n",
+    "        self.input_query_textarea = widgets.Textarea(\n",
+    "            value='',\n",
+    "            placeholder='<INSERT QUERY>',\n",
+    "            description='Query:',\n",
+    "            layout=wu.input_layout,\n",
+    "            style=wu.input_style\n",
+    "        )\n",
+    "        self.query = None\n",
+    "        self.query_result = None\n",
+    "        self.output = widgets.Output()\n",
+    "        self.resolve_button = wu.StyledButton(\n",
+    "            \"Resolve resource\", \"Click to resolve a referenced resource\", \"check\",).get()\n",
+    "        self.query_button = wu.StyledButton(\n",
+    "            \"Query resource\", \"Click to query your referenced resource\", \"check\",).get()\n",
+    "        self.resolve_button.on_click(self.resolve_resource)\n",
+    "        self.query_button.on_click(self.run_query)\n",
+    "        self.vb = widgets.VBox(\n",
+    "            children=[\n",
+    "                self.label,\n",
+    "                self.name_dropdown,\n",
+    "                self.resolve_button,\n",
+    "                self.output\n",
+    "            ],\n",
+    "            layout=wu.vbox_layout\n",
+    "        )\n",
+    "\n",
+    "    def get_bq_resources(self):\n",
+    "        result = subprocess.run(\n",
+    "            [\"terra\", \"resource\", \"list\", \"--format=JSON\"], capture_output=True, text=True)\n",
+    "        ids_list = wu.list_bq_tables(result.stdout)\n",
+    "        return ids_list\n",
+    "\n",
+    "    def run_query(self, b):\n",
+    "        with self.output:\n",
+    "            print(\n",
+    "                f\"Accessing table '{self.table.project}.{self.table.dataset_id}.{self.table.table_id}'.\")\n",
+    "            self.query = self.input_query_textarea.value\n",
+    "            print(f\"Running query {self.query}\")\n",
+    "            print(\"\")\n",
+    "            self.query_result = pd.read_gbq(self.query)\n",
+    "            print(\"Result...\")\n",
+    "            print('')\n",
+    "            print(self.query_result)\n",
+    "\n",
+    "    def resolve_resource(self, b):\n",
+    "        with self.output:\n",
+    "            commandList = [\"terra\", \"resolve\",\n",
+    "                           f\"--name={self.name_dropdown.value}\"]\n",
+    "\n",
+    "            print('Running command:')\n",
+    "            print('\\n'.join(commandList))\n",
+    "            print('')\n",
```

Comment:
![image](https://github.com/DataBiosphere/terra-axon-examples/assets/6879774/0dabb195-e34b-4921-b4d6-fd777750a19d)
This outputs the commands one word per line.
Is it possible to have it so the output looks like
```
terra resolve \
  --name=bigquery-public-data.human_genome_variants.1000_genomes_pedigree
```

Reply:
updated to look like [this](https://screenshot.googleplex.com/59at8kk6nmhem9K)